src/App.js

```diff
@@ -0,0 +1,7 @@
+function App() {
+  return (
+    <div>Working</div>
+  );
+}
+
+export default App;
```

Comment:
**REQUIRED**
* Let's delete this since we have an App component in `src/components`.